### PR TITLE
Fetch streaming requests: Update article for Chrome 105

### LINKED ISSUE #8288: content: Streaming requests with the fetch API seemly not work
### What page(s) need to be updated?

[fetch-upload-streaming](https://web.dev/fetch-upload-streaming)

### Why is this update needed?

The article describes that Request can use Stream as a body in Chrome 95, but I opened the demo in subsequent versions  and was told that the current browser does not support this feature

Also, [caniuse](https://caniuse.com/mdn-api_request_request_readablestream_request_body) says that any version does not support this feature. However, I don't see the article revealing that this feature requires a specific flag to be enabled.

### PR DESCRIPTION
**Pull request by jakearchibald**
Fixes #8288

Preview: https://deploy-preview-8317--web-dev-staging.netlify.app/fetch-upload-streaming/

Changes proposed in this pull request:

- Clarifies that feature is launching in Chrome 105
- Removes origin trial details, and other references to experimentation
- Updates API references and feature detect
- Reordering article a bit, as some of the restrictions became more important
- Updated details on duplexing, since it's a new option
- Removed references to the HTTP/1 opt-in, as it's no longer a thing
- Changed the hero image because it was low contrast and it annoyed me

### CODE REVIEW

File: src/site/content/en/blog/fetch-upload-streaming/index.md [hunk 1]
```diff
@@ -19,18 +19,21 @@ feedback:
   - api
 ---
 
-From Chrome 95, you can start a request before you have the whole body available by using the streams API.
+From Chrome 105, you can start a request before you have the whole body
```
Comment (tomayac) [inline src/site/content/en/blog/fetch-upload-streaming/index.md:22]:
```suggestion
From Chromium&nbsp;105, you can start a request before you have the whole body
```

**Comment by jakearchibald** [comment]
@rachelandrew fwiw, this should probably be on d.c.c

**Review by tomayac — APPROVED** [review]
LGTM, just a nit for Chrome → Chromium (on web.dev).

**Comment by tomayac** [comment]
(If it goes out on d.c.c., we can purely talk about Chrome of course.)

**Comment by rachelandrew** [comment]
@jakearchibald @tomayac yeah this should get moved. It's not a good fit here.

**Comment by jakearchibald** [comment]
Should I open a PR over on d.c.c and you can set up the redirect? (I don't know how to do that bit)

**Comment by rachelandrew** [comment]
@jakearchibald sounds great - I can sort the redirect

**Comment by nwtgck** [comment]
Hi, thank you so much for the update!

> The fetch will be rejected if the connection isn't HTTP/2.

I'm not sure this is right. In my experiment HTTP/3 connection was allowed in Version 105.0.5148.2 (Official Build) dev (x86_64) as follows.

image

**Comment by nwtgck** [comment]
@jakearchibald Thank you for the update!

**Comment by jakearchibald** [comment]
Moved to https://github.com/GoogleChrome/developer.chrome.com/pull/3214

**Pull request closed without merging**